.travis-build-without-test.sh

```diff
@@ -7,6 +7,11 @@ export SHELLOPTS
 
 SLUGOWNER=${TRAVIS_REPO_SLUG%/*}
 
+# Optional argument $2 is one of:
+#   jdk7, jdk8
+# If omited, then either jdk may be used.
```

Comment:
I'm not sure what "may be used" means.  Is this a desirable amount of flexibility for the script, and is it useful for people?  Would it be better to set a default?